.gitlab-ci.yml

```diff
@@ -57,6 +57,15 @@ test_bulk_update:
   tags:
     - ingest-integration-tests
 
+test_direct_archiving:
+  stage: test
+  script:
+     - python -m unittest tests.test_ingest.TestRun.test_direct_archiving
```

Comment:
Why is the package called "test_ingest" it is very general. All the tests test ingest here. I suggest to put tests in packages according to the module or feature being testsed.

Reply:
`TestIngest` is a class. I am not sure it would worth to create modules for only 1 class per module. We are only talking about 5 classes all together.